security/README.md

```diff
@@ -1,7 +1,9 @@
 # Security
 
-Security issues can easily arise through incorrectly built front end code.
+All web developers should be aware of security risks related to web technologies.
 
-* [Secure markup](secure-markup.md)
+* The Open Worldwide Application Security Project ([OWASP](https://owasp.org/)) is a nonprofit foundation that works to improve the security of software. They maintain a [top 10 list of critical security risks in web applications](https://owasp.org/www-project-top-ten/), which is essential reading.
+* Mozilla's open-source infosec documentation contains an excellent [Web Security cheat sheet](https://infosec.mozilla.org/guidelines/web_security.html) we encourage you to read.
```

Comment:
Being picky here but I think Cheat Sheet should be capitalised here?

Reply:
And maybe "that we encourage you to read" (added "that").

Reply:
Exceptionally picky! but correct on the capitalisation, thanks :)

I disagree with "that we encourage you to read" though.